modules/microshift-ingress-controller-config.adoc

```diff
@@ -0,0 +1,153 @@
+
+[NOTE]
+====
+* After you create it, the `config.yaml` file takes precedence over built-in settings. The configuration file is read every time the {microshift-short} service starts.
+* Configuration snippet YAMLs take precedence over all settings anda `config.yaml` configuration file. See the Additional resources links for more information.
+====
+
+. When using the `config.yaml`, replace the default values in the `network` section of the {microshift-short} YAML with your valid values.
++
+.Ingress controller configuration field snippet with default values
+[source,yaml]
+----
+apiServer:
+# ...
+ingress:
+    defaultHTTPVersion: 1
+    forwardedHeaderPolicy: Append
+    httpCompression:
+        mimeTypes:
+            - ""
+    httpEmptyRequestsPolicy: Respond
+# ...
+    logEmptyRequests: Log
+# ...
+    tuningOptions:
+        clientFinTimeout: 1s
+        clientTimeout: 30s
+        headerBufferBytes: 0
+        headerBufferMaxRewriteBytes: 0
+        healthCheckInterval: 5s
+        maxConnections: 0
+        serverFinTimeout: 1s
+        serverTimeout: 30s
+        threadCount: 4
+        tlsInspectDelay: 5s
+        tunnelTimeout: 1h
+# ...
+----
+
+[cols="3a,8a",options="header"]
+|===
+|Parameter |Description
+
+|`defaultHTTPVersion`
+|`defaultHTTPVersion` sets the HTTP version for the Ingress Controller. Default value is `1`.
+//Q: which versions do we support; HTTP/1.0, 2 and 3? do we need to configure a load balancer for 2 and 3?
+
+|`forwardedHeaderPolicy`
+|By setting the `forwardedHeaderPolicy`, you specify when and how the Ingress Controller sets the `Forwarded`, `X-Forwarded-For`, `X-Forwarded-Host`, `X-Forwarded-Port`, `X-Forwarded-Proto`, and `X-Forwarded-Proto-Version` HTTP headers. The following values are valid:
+
+* `Append`, preserves any existing headers by specifying that the Ingress Controller appends them.
+* `Replace`, removes any existing headers by specifying that the Ingress Controller sets the headers.
+* `IfNone` sets the headers set by specifying that the Ingress Controller sets the headers  if they are not already set.
+* `Never`, preserves any existing headers by specifying that the Ingress Controller never sets the headers.
+
+|`httpCompression`
+|`httpCompression` defines the policy for HTTP traffic compression.
+
+* `httpCompression.mimeTypes` defines a list of or a single MIME type to which compression is applied. For example, `text/css; charset=utf-8`, `text/html`, `text/*`, `image/svg+xml`, `application/octet-stream`, `X-custom/customsub`, using the format pattern, `type/subtype; [;attribute=value]`. The `types` are: application, image, message, multipart, text, video, or a custom type prefaced by `X-`. To see the full notation for MIME types and subtypes, see link:https://datatracker.ietf.org/doc/html/rfc1341#page-7[RFC1341]
```

Comment:
Also, per the [docs guidelines](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#lead-in-link-sentence-style-for-the-link-attribute), when linking to external sites we should add a description of the location in parentheses following the link: 
```suggestion
* `httpCompression.mimeTypes` defines a list of or a single MIME type to which compression is applied. For example, `text/css; charset=utf-8`, `text/html`, `text/*`, `image/svg+xml`, `application/octet-stream`, `X-custom/customsub`, using the format pattern, `type/subtype; [;attribute=value]`. The `types` are: application, image, message, multipart, text, video, or a custom type prefaced by `X-`. To see the full notation for MIME types and subtypes, see link:https://datatracker.ietf.org/doc/html/rfc1341#page-7[RFC1341] (IETF Datatracker documentation).
```

I'm guessing the site description/name here, so please change if needed